app/App.hs

```diff
@@ -15,39 +12,46 @@ import PSR.ContextBuilder
 import PSR.Streaming
 import Streamly.Data.Fold.Prelude qualified as Fold
 import Streamly.Data.Stream.Prelude qualified as Stream
-import System.Exit (exitFailure)
+import Text.Pretty.Simple
 
 --------------------------------------------------------------------------------
 -- Main
 --------------------------------------------------------------------------------
 
+compactPrintOpts :: OutputOptions
+compactPrintOpts =
+    defaultOutputOptionsDarkBg
+        { outputOptionsCompact = True
+        , outputOptionsCompactParens = True
+        , outputOptionsIndentAmount = 2
+        , outputOptionsStringStyle = Literal
+        }
+
 main :: IO ()
 main = do
     Options{..} <- execParser psrOpts
-    CM.ConfigMap{..} <- decodeFileThrow scriptYaml
-    -- TODO: Remove
-    -- TODO: Is there really no better way to parse a hash???
-    hash <- case C.deserialiseFromJSON "\"082efb838a6a38c435bf5a1c823569ad224fab42034d265b95f88665ee27877f\"" of
-        Left err -> do
-            print err
-            exitFailure
-        Right h -> pure h
-    let points = maybe [C.ChainPoint 173166134 hash] pure start -- [ChainPointAtGenesis]
+    config@CM.ConfigMap{..} <- either error pure =<< CM.readConfigMap scriptYaml
+
+    -- TODO: What's a better default here?
+    let points = maybe [C.ChainPointAtGenesis] pure cmStart
```

Comment:
`ChainPointAtGenesis` will be too old in most cases unless we start the app as soon as we run the testnet.
An alternative default can be the `ChainTip` (which we can query).